website/content/docs/auth/jwt.mdx

```diff
@@ -0,0 +1,136 @@
+---
+layout: docs
+page_title: JWT - Auth Method
+description: >-
+  The JWT auth method allows authentication using signed JWTs
+---
+
+# JWT auth method
+
+@include 'x509-sha1-deprecation.mdx'
+
+The `jwt` auth method can be used to authenticate with Vault using JSON Web Tokens
+([JWT](https://en.wikipedia.org/wiki/JSON_Web_Token)). This form of authentication
+is commonly used in Workload Identity Federation (WIF) workflows for machine-to-machine
+authentication.
+
+The auth method is configured to trust a JWT issuer. For example, this could be a
+cloud service provider that makes workload identity available in a virtual machine
+or CI provider that provides identity to jobs. It must be [configured](#jwt-verification)
+with public keys from the JWT issuer by either OIDC Discovery, a JSON Web Key Set (JWKS) URL,
+or locally-provided keys. A signed JWT from the configured issuer is required as the form
+of authentication. The auth method will cryptographically verify the signature using the
+issuer's public key. The verified JWT claims can be used to provide [further validation](#bound-claims)
+to gate authentication to Vault.
+
+## Authentication
+
+### Via the CLI
+
+```shell-session
+$ vault write auth/jwt/login role=demo jwt=...
+```
+
+The default path for the JWT authentication backend is `/jwt`, so if you're using the
+default backend, the command would be:
+
+```shell-session
+$ vault write auth/jwt/login role=demo jwt=...
+```
+
+If your JWT auth backend is using a different path, use that path.
+
+### Via the API
```

Comment:
```suggestion
### Authenticate with the API
```

Style correction: Use directive language for headings, make headings friendly to screen readers